### DISCUSSION TITLE
Extending renovate slim docker image

### DISCUSSION THREAD

**Discussion by Tobias-08** [renovatebot/renovate#16427] [Request Help]
Is it possible to extend the slim image with only one manager (in this case dotnet) without the need of publishing an own image (disclaimer: I'm Docker-newbie)? That means:

- Starting slim image
- install dotnet ->What options does the slim image offer for doing this (apt-get...?)?
- execute renovate

Thanks!

**Comment by viceice** [comment]
it depends on how are you running the docker image.

**Reply by Tobias-08** [reply]
As a container job in Azure Devops.

**Reply by viceice** [reply]
that's not enough information, we don't know / use that. so please provide additional information.

you need to check if you can run custom commands or if it's possible to build a image before running it without publish to registry. that's nothing renovate can influence.

**Reply by Tobias-08** [reply]
OK, sorry.

- I am able to run custom commands before running renovate. But I don't know which options are available to install something in the slim image.
- I am not aware of an option to run an unpublished image in Azure Devops.
